README.md

```diff
@@ -212,6 +212,8 @@ Note: this is helpful if your application sits behind a proxy (or set of proxies
 
 ChangeLog
 ---------
+* **1.7.1**
```

Comment:
The only reason I thought it should still be 1.7.x was that it's backwards compatible (i.e. you can still use `extractIps` instead of `getIdentifiers`). Happy to bump it up if you think it still should be.